src/App.css

```diff
@@ -93,6 +93,26 @@ a:visited {
   font-size: 1.1rem;
 }
 
+.nonPurchasedItem {
+  list-style-type: none;
+  padding: 10px;
+  margin: 10px;
+  background-color: #ff165d;
+  border-radius: 8%;
+}
+
+.nonPurchasedItem:hover,
+.purchasedItem {
+  list-style-type: none;
+  padding: 10px;
+  margin: 10px;
+  background-color: #ff165d;
+  border-radius: 8%;
+  text-decoration: line-through;
+  cursor: pointer;
+  opacity: 80%;
+}
+
```

Comment:
![bitmoji](https://render.bitstrips.com/v2/cpanel/ed555e68-6e66-434d-82fd-cf2515e997b7-141935a4-509e-461c-bba3-4228c119c4d3-v1.png?transparent=1&palette=1&width=100)

Awesome - I like it! 👏

Reply:
Thanks!